skins/oasis/modules/templates/Oasis_Index.php

```diff
@@ -74,6 +74,7 @@
 <?= $dynamicYield ?>
 <?= $ivw2 ?>
 <?= $rubiconRtp ?>
+<gcse:search></gcse:search>
```

Comment:
shouldn't this be included in a template only if `wgGoogleSearchTest` is set to true?